docs/resources/dli_flinkjob_jar.md

```diff
@@ -0,0 +1,150 @@
+---
+subcategory: "Data Lake Insight (DLI)"
+---
+
+# huaweicloud_dli_flinkjar_job
+
+Manages a flink job resource which type is `Flink Jar` within HuaweiCloud DLI.
+
+## Example Usage
+
+### Create a flink job
+
+```hcl
+variable "name" {}
+variable "queue_name" {}
+variable "jar_obs_path" {}
+variable "entrypoint_args" {}
+
+resource "huaweicloud_dli_package" "test" {
+  group_name  = "jarPackage"
+  type        = "jar"
+  object_path = var.jar_obs_path
+}
+
+resource "huaweicloud_dli_flinkjar_job" "test" {
+  name            = var.name
+  queue_name      = var.queue_name
+  entrypoint      = "${huaweicloud_dli_package.test.group_name}/${huaweicloud_dli_package.test.object_name}"
+  entrypoint_args = var.entrypoint_args
+
+  tags = {
+    foo = "bar"
+    key = "value"
+  }
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the DLI flink job resource. If omitted, the
+  provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the name of the job. Length range: 1 to 57 characters.
+ Which may consist of letters, digits, underscores (_) and hyphens (-).
+
+* `description` - (Optional, String) Specifies job description. Length range: 1 to 512 characters.
+
+* `queue_name` - (Optional, String) Specifies name of a queue. The type of queue must be `general`.
```

Comment:
The type of queue must be `general`. --- I think it is hard to understanding as the **type** means `queue_type` of huaweicloud_dli_queue.